gsitemap.php

```diff
@@ -260,7 +266,7 @@ public function emptySitemap($id_shop = 0)
      */
     public function addLinkToSitemap(&$link_sitemap, $new_link, $lang, &$index, &$i, $id_obj)
     {
-        if ($i <= 25000 && memory_get_usage() < 100000000) {
+        if ($i <= 49000 && memory_get_usage() < 1000000000) {
```

Comment:
Could you explain the reason to increase these values?

Reply:
my eshop have 40k+ products, if $i is 25000 i get ~half of products.

Reply:
Not sure it's the right way to do it.
Maybe people doesn't have the amount of products you have, and the server quality you have :thinking: 
We maybe need to understand why the memory_get_usage has been added.
Also, I think if you have a lot of products, you should use cron task to do it :)

Reply:
I have dedicated server for this eshop only. memory usage for that amount of product and two languages is more than 100mb(and fails to generate XML). 

I have test with 200mb and its ok.

Reply:
keep original value